epocas-backoffice/app/Controllers/Http/OccasionController.js

```diff
@@ -91,6 +91,27 @@ class OccasionController {
 
     await occasion.delete();
   }
+
+  async current({ request, response }) {
+    const occasions = await Occasion.all();
+    const date = Date.now();
+    let current = -1;
```

Comment:
Porque é que declaraste como -1 aqui?

Reply:
Como diz na descrição do PR: "If there's no current occasion, it returns -1"